cardano-ledger-test/src/Test/Cardano/Ledger/Examples/TwoPhaseValidation.hs

```diff
@@ -1889,3 +1929,29 @@ alonzoBBODYexamples =
     ]
   where
     pf = Alonzo Mock
+
+testEvaluateTransactionFee :: Assertion
+testEvaluateTransactionFee =
+  evaluateTransactionFee @A
+    pparams
+    validatingTxNoWits
+    1
+    @?= minfee pparams (validatingTx pf)
```

Comment:
Same here, without providing expected value we aren't testing the implementation of `minfee`, because `evaluateTransactionFee` itself call `minfee`. In other words if `minfee` has a bug`, we won't catch it. Hard coding expected values in tests is better.

Reply:
maybe I should just have both tests?

Reply:
I can add a property test just like that in the work that I am doing. So I can change this test to a hard coded value in a later PR, if that works for you.

Reply:
that would be great, thank you!